src/examples/PdfDownload/PdfDownload.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { PdfDownload } from "./PdfDownload";
+import { pdfDownloadCode } from "./PdfDownload.code";
+
+<Meta title="Examples/PdfDownload" />
+
+# Simple wizard example
```

Comment:
seems this example still just a copy of the simple wizard example, can you use this example to show the pdf function?